sql/src/main/java/org/apache/druid/sql/calcite/expression/builtin/IPv4AddressMatchOperatorConversion.java

```diff
@@ -49,7 +49,7 @@ public class IPv4AddressMatchOperatorConversion extends DirectOperatorConversion
               SUBNET_OPERAND
           )
       )
-      .returnTypeInference(ReturnTypes.BOOLEAN_NULLABLE)
+      .returnTypeNullable(SqlTypeName.BOOLEAN)
```

Comment:
should we add a test for this one too?

Reply:
This does not need to be changed as ipv4 even if 2nd arg is null throws a validation exception and does not return null in anyway. So I think it can be left out